dcos/ssh_util.py

```diff
@@ -0,0 +1,131 @@
+import os
```

Comment:
Could you add a new line after this? I don't think we have a linter rule for this, but the current codebase tries to stick to a PEP convention which recommends using 3 blocks for imports (standard, external libraries, and locals, see http://legacy.python.org/dev/peps/pep-0008/#imports).